readme.md

```diff
@@ -1,31 +1,37 @@
-# Demo project
+# Upsun demo
 
-This is a simple demo project meant to take a user on a bit of a product tour. 
+This repository is the Upsun demo project meant to help you learn the basics of [Upsun PaaS](https://upsun.com/).
 
-## Testing the demo on Upsun
+## Run de demo on Upsun
 
-### Part 1: Replicating what is provided in Console
+### 1. Deploy the demo
 
-1. Clone the repository, and create an organization and a project on Upsun we'll deploy it to:
+1. Clone the GitHub repository locally:
 
 - ```
   git clone git@github.com:platformsh/demo-project.git upsun-demo && cd upsun-demo
   ```
+
+2. Create a new Upsun organization if you do not have one yet:
+
 - ```
-  upsun organization:create --label "Upsun Testing" --name upsun-testing
-  ```
-- ```
-  upsun create --org upsun-testing --title "Upsun demo" --region "org.recreation.plat.farm" --plan flexible --default-branch main --no-set-remote -y
+  upsun organization:create
   ```
 
-2. Set the remote for the project, and first push:
+3. Create a new Upsun project:
 
 - ```
-  PROJECT_ID=$(upsun project:list --title "Upsun demo" --pipe)
+  upsun project:create
   ```
+
+4. Set the `upsun` Git remote:
```

Comment:
I would remove this step and set `--set-remote` above (or even without adding the flag, the interactive mode asks the question, right now the default answer is N, but I suppose it should be Yes).

Reply:
https://github.com/platformsh/legacy-cli/pull/1325

Reply:
Thank you @pjcdawkins 
This step can definitely be removed now :)